class/DataProvider/Student/LocalDbStudentDataProvider.php

```diff
@@ -92,21 +84,8 @@ protected function plugValues(&$student, Array $data)
             $student->setCampus($data['campus']);
         }
 
-        // Level (grad vs undergrad)
-        // TODO: Merge this with changes for variable levels
-        if($data['level'] == self::UNDERGRAD) {
-            $student->setLevel(self::UNDERGRAD);
-        } else if ($data['level'] == self::GRADUATE) {
-            $student->setLevel(self::GRADUATE);
-        } else if ($data['level'] == self::GRADUATE2) {
-            $student->setLevel(self::GRADUATE2);
-        } else if ($data['level'] == self::DOCTORAL) {
-            $student->setLevel(self::DOCTORAL);
-        } else if ($data['level'] == self::POSTDOC) {
-            $student->setLevel(self::POSTDOC);
-        } else {
-            $student->setLevel(null);
-        }
+        // Level
+        $student->setLevel($data['level']);
```

Comment:
If $data['level'] is an unknown, how is it handled? Previously, it was set to null. With your change, $data['level'] could be an undefined term and will set it. Will that affect the code?